packages/astro/test/units/vite-plugin-scanner/scan.test.js

```diff
@@ -0,0 +1,64 @@
+import { expect } from 'chai';
+import { scan } from '../../../dist/vite-plugin-scanner/scan.js';
+
+describe('astro scan', () => {
+	it('should return empty object', async () => {
+		const result = await scan(`export {}`, '/src/components/index.astro');
+		expect(Object.keys(result).length).to.equal(0);
+	});
+
+	it('recognizes constant boolean literal (false)', async () => {
+		const result = await scan(`export const prerender = true;`, '/src/components/index.astro');
+		expect(result.prerender).to.equal(true);
+	});
+
+	it('recognizes constant boolean literal (false)', async () => {
+		const result = await scan(`export const prerender = false;`, '/src/components/index.astro');
+		expect(result.prerender).to.equal(false);
+	});
+
+	it('recognizes let boolean literal', async () => {
+		const result = await scan(`export let prerender = true;`, '/src/components/index.astro');
```

Comment:
I thought we were going to make this only work with `const`. Using let someone might think they could do:

```js
export let prerender = true;

if(someCondition) {
  prerender = false;
}
```

Reply:
Yes, we did agree to that! I avoided doing it here because there wasn't an easy way to surface what type of variable you were using, but I'm sure I can figure something out.

Reply:
Updated in [d1546a2](https://github.com/withastro/astro/pull/5297/commits/d1546a229436c2642b4c59e1092bde1019036964)